src/Illuminate/Broadcasting/Broadcasters/AblyBroadcaster.php

```diff
@@ -139,62 +193,114 @@ public function broadcast(array $channels, $event, array $payload = [])
     }
 
     /**
-     * Build an Ably message object for broadcasting.
+     * @param $channelName string
+     * @param $token string
+     * @param $clientId string
+     * @param $channelCapability string[]
+     * @return string
+     */
+    public function getSignedToken($channelName, $token, $clientId, $channelCapability = ['*'])
+    {
+        $header = [
+            'typ' => 'JWT',
+            'alg' => 'HS256',
+            'kid' => $this->getPublicToken(),
+        ];
+        // Set capabilities for public channel as per https://ably.com/docs/core-features/authentication#capability-operations
+        $channelClaims = $this->defaultChannelClaims;
+        $serverTimeFn = function () {
+            return self::getServerTime();
+        };
+        if ($token && Utils::isJwtValid($token, $serverTimeFn, $this->getPrivateToken())) {
+            $payload = Utils::parseJwt($token)['payload'];
+            $iat = $payload['iat'];
+            $exp = $payload['exp'];
+            $channelClaims = json_decode($payload['x-ably-capability'], true);
+        } else {
+            $iat = $serverTimeFn();
+            $exp = $iat + $this->tokenExpiry;
+        }
+        if ($channelName) {
+            $channelClaims[$channelName] = $channelCapability;
+        }
+        $claims = [
+            'iat' => $iat,
+            'exp' => $exp,
+            'x-ably-clientId' => $clientId ? strval($clientId) : null,
+            'x-ably-capability' => json_encode($channelClaims),
+        ];
+
+        return Utils::generateJwt($header, $claims, $this->getPrivateToken());
+    }
+
+    /**
+     * Remove prefix from channel name.
      *
-     * @param  string  $event
-     * @param  array  $payload
-     * @return \Ably\Models\Message
+     * @param $channel string
```

Comment:
```suggestion
     * @param  string  $channel
```
First variable type and then the variable name

Reply:
Fixed!